lib/team.rb

```diff
@@ -0,0 +1,13 @@
+
+class Team
+    attr_reader :team_id, :franchiseId, :teamName, :abbreviation,
+                :stadium
+
+    def initialize(team_info)
+        @team_id = team_info[:team_id]
+        @franchiseId = team_info[:franchiseId]
```

Comment:
Love the set up for the hash. We probably will need to convert some to intergers to manipulate the data later